src/modules/InviteUser/components/InviteUser.vue

```diff
@@ -52,29 +61,39 @@ export default {
       }
     },
     async handleSubmit() {
-      
-      if (!this.name.value || !this.email.value || !parseInt(this.role_id.value)) {
+      if (this.submission.isVerified){
```

Comment:
What is the purpose of if (this.submission.isVerified) ?
- Is "!" mark missing?

Reply:
the  purpose of if (this.submission.isVerified) is to stop reloading the handleSubmit() function after it is already verified

Reply:
the reloading occured on line 86:  document.getElementById("inviteBtn").click(); 
this line recalls the handleSubmit() Function

Reply:
the purpose of if (this.submission.isVerified) is to stop reloading the handleSubmit() function after it is already verified